docs/08_getting_started.md

```diff
@@ -1,7 +1,110 @@
 # Getting Started
-This section provides you with the steps that are required to use the extractor.
+This section provides you with the steps that are required to use the extractor in your android app.
 
-### Implement a Downloader
+
+## Configure project.
+### Add dependencies to gradle:
+
+1. **project** `settings.gradle`:
+
+```gradle
+...
+buildscript {
+    repositories {
+        ...
+        maven { url 'https://jitpack.io' }
+    }
+    ...
+}
+```
+
+
+
+2. module/app build.gradle:
+
+```gradle
+// NewPipe Libraries
+implementation 'com.github.TeamNewPipe:NewPipeExtractor:0.21.12'
+implementation 'com.github.TeamNewPipe:nanojson:1d9e1aea9049fc9f85e68b43ba39fe7be1c1f751'
+
+
+// HTTP client
+implementation "com.squareup.okhttp3:okhttp:3.12.13"
+```
+
+Note that you could use any http client other then okhttp, but it used here for more simplicity.
+
+
+### Make sure you use Java 11
```

Comment:
Does the extractor really have the java 11 requirement? I thought that it didn't even support the full Java 8 feature set.

Reply:
For some reason I wasn't able to run it below java 8 for android, I really don't know why..

However this might change when we find a solution ;)

Reply:
Also this is what NewPipe itself does!

See https://github.com/TeamNewPipe/NewPipe/blob/dev/app/build.gradle#L82

Reply:
For 'normal' java apps, it seems to run fine on java 8, see
https://github.com/cloneconf/newpipe-java-example/blob/c90332e7e34365d2424bb16697f88fbdd784f737/build.gradle#L13  ([full example](https://github.com/cloneconf/newpipe-java-example))